source/egg/math/Math.cc

```diff
@@ -280,6 +326,20 @@ f32 frsqrt(f32 x) {
     return tmp1 * tmp2;
 }
 
+f32 AtanFIdx_(f32 x) {
+    u16 idx;
+    f32 val;
+    f32 r;
+
+    x *= 32.0f;
+    idx = static_cast<u16>(x);
+    r = x - static_cast<f32>(idx);
+
+    val = sArcTanTbl[idx].atanVal + r * sArcTanTbl[idx].atanDt;
```

Comment:
Is it fine that we're indexing into the array twice at the same index? Can we retrieve it to a const ref first and just use that twice? idk if it matters as the compiler will catch it

Reply:
The compiler will catch it.